docs/style.md

```diff
@@ -0,0 +1,28 @@
+# Style Guide
+
+## Generic
+
+* This guide can be ignored for data files that are not to be viewed in an editor.
+* 120 character per line for all source files.
+* Avoid white space errors like trailing spaces.
+
+
+## Python Code
+
+* We mostly follow [PEP8](https://peps.python.org/pep-0008/). Read below to see the differences.
+* 120 characters per line. PyCharm does this automatically, other editors can be configured for it.
+* Strings in core code will be `"strings"`. In other words: double quote your strings.
```

Comment:
Why enforce double quotes? if a string uses double or single quote, your editor will color it as string, doesnt matter which veriant you use

Reply:
To make the **core** code consistent. This was discussed in discord and the majority was for using `"`

Reply:
i do prefer double quotes but i hate making it a rule, single quotes have thier use, like if you want to escape a double quote
like 'Received Item:: "{item.name}"'

Reply:
Python agrees with you on that one
```
>>> "\"test\""
'"test"'
>>> '\'test\''
"'test'"
```
I don't think we have specified what to do in that case, and I doubt a PR gets rejected for `"\"` or `'"'`.